app/controllers/inventory_food_controller.rb

```diff
@@ -0,0 +1,42 @@
+class InventoryFoodController < ApplicationController
+  def index
+    @inventory_food = InventoryFood.all
+  end
+
+  def show
+    @inventory_food = InventoryFood.find(params[:id])
+  end
+
+  def new
+    @inventory_food = InventoryFood.new
+    @inventory = Inventory.find(params[:inventory_id])
+  end
+
+  def create
+    @inventory_food = InventoryFood.create(inventory_food_params)
+    @inventory_food.inventory_id = params[:inventory_id]
+    @inventory_food.food_id = params[:food_id]
+    @inventory_food.save
+    redirect_to user_inventory_path(current_user.id, params[:inventory_id]),
+                notice: 'Inventory Food was successfully created.'
+  end
+
+  def destroy
+    @inventory_food = InventoryFood.find_by(id: params[:id])
+
+    redirect_to user_inventory_path(current_user.id, params[:inventory_id]),
+                notice: 'Inventory Food was successfully deleted.'
+
```

Comment:
- [ ] it would be better if you Shows the list of food that is missing by comparing the food in the recipe with the food in the inventory as required in order to complete the requirements 
<img width="1024" alt="Screenshot 2022-12-23 at 17 46 56" src="https://user-images.githubusercontent.com/61790587/209371745-0c407ac9-44a2-48ef-92e3-5ab43e6a8c04.png">